package.json

```diff
@@ -30,6 +30,7 @@
  "devDependencies": {
   "@babel/core": "^7.7.4",
   "@babel/preset-env": "^7.7.4",
+  "@nextcloud/browserslist-config": "^1.0.0",
```

Comment:
maybe use it? :p

Reply:
mimi

Reply:
I'll do it in the rebase of #153

Reply:
I added it because the code wouldn't build otherwise